tools/cmake/toolchain/openwrt.cmake

```diff
@@ -7,7 +7,9 @@ set(CMAKE_CROSS_COMPILING ON)
 
 # Platform specific
 set(TARGET_PLATFORM             "openwrt" CACHE STRING "Target Platform")
+if (DEFINED ENV{PRPLMESH_PLATFORM_TYPE})
 set(TARGET_PLATFORM_TYPE        "$ENV{PRPLMESH_PLATFORM_TYPE}" CACHE STRING "Target Platform Type")
+endif()
```

Comment:
Why is this condition needed? If TARGET_PLATFORM_TYPE is not defined in the environment, it will just be defined to empty string and that's OK, no?

Reply:
I don't know cmake, so trying to stay safe, I'll try what happens if it's left undefined and if it works  remove the condition.